hazelcast/config.py

```diff
@@ -1385,3 +1386,14 @@ def from_dict(cls, d):
                     "Unrecognized config option for the flake id generator: %s" % k
                 )
         return config
+
+
+def config_dict(**kwargs) -> six.types.Dict[str, six.types.Any]:
+    """Create a valid dictionary of HazelcastClient configuration options."""
```

Comment:
This isn't mean to replace the current way of configuring the client, but to complement it.
```python
# both of these would still be valid

HazelcastClient(foo=bar, fizz=buzz, xml="my_settings.xml")

my_validated_config = config_dict(foo=bar,fizz=buzz, xml="my_settings.xml")
HazelcastClient(**my_validated_config)

```
The difference is I can be sure that my configuration settings are valid before I attempt to instantiate my client, and I can see how my configuration settings resolve when I'm combining XML and kwarg settings.